test/e2e/case2_aggregation_test.go

```diff
@@ -15,13 +15,13 @@ import (
 	"open-cluster-management.io/governance-policy-propagator/test/utils"
 )
 
-const (
-	case2PolicyName string = "case2-test-policy"
-	case2PolicyYaml string = "../resources/case2_aggregation/case2-test-policy.yaml"
-)
-
 var _ = Describe("Test policy status aggregation", func() {
```

Comment:
Have you tested the unknown case here

Reply:
Not here, but it is tested in the unit-style test `TestCalculateRootCompliance`